.github/scripts/boost/build_and_test_boost.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Please split the script to a separate ones for the build and test phase so it can be used separately both locally and as a separate job steps on GitHub Actions.